### PR TITLE
Add StackStorm Copyright headers

### PR DESCRIPTION
**Pull request by armab**
Add new 2020 StackStorm project copyright headers which operates now under the Linux Foundation. “StackStorm Authors” accumulates all the contributions made by the new StackStorm Open Source project maintainers, contributors and community.
This new copyright string should be added on top of existing ExtremeNetworks copyright header.

References
* https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/
* https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 The StackStorm Authors.
```
Comment (armab) [inline LICENSE:190]: Perfect find! :+1:

**Comment by armab** [comment]
@nmaludy We'll need a PR approval to merge this one.

**Comment by armab** [comment]
Looks like the changes in this PR breaks Unit tests for both py2 and py3 builds in some weird way.

The CI log is legit and build failure is consistent between the retries: https://travis-ci.org/github/StackStorm/st2/jobs/716997316#L3401


However I can't find any change that may result in such error and `test_sensor_with_typo.py` file exists.
@nmaludy do you see :eyes: anything suspicious?

**Comment by nmaludy** [comment]
@armab looks like this unit test had hard coded line number in it, when i inserted a new line in there it was off by one. just pushed a new commit that fixes it to the new line number, hopefully we're good now.

**Comment by armab** [comment]
@nmaludy I missed the line number. Thanks!

**Pull request merged**